### PR TITLE
Fix FAQ page titles

### PR DESCRIPTION
**Pull request by lnhsingh**
Fix CockroachDB FAQ page title. Relates to DOC-2377.

### CODE REVIEW

File: v21.1/frequently-asked-questions.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Frequently Asked Questions
+title: CockroachDB Frequently Asked Questions
```
Comment (stbof) [inline v21.1/frequently-asked-questions.md:2]:
```suggestion
title: CockroachDB FAQs
```

**Review by stbof — COMMENTED** [review]
See my suggestions and comments.

**Review by lnhsingh — COMMENTED** [review]
Yeah, changing to the acronym makes sense to me. I suppose the only downside is that we don't define what "FAQ" stands for, but not sure if we need to since it's so ubiquitous.  I changed the summary a bit, just in case

Updated "Multi-Active Availability" to "Availability FAQs"

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13596#-:-N-YpO1qEp2Rtp3WwifX:b8g1wsa)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stbof)

___
*[v21.1/frequently-asked-questions.md, line 2 at r1](https://reviewable.io/reviews/cockroachdb/docs/13596#-N-YLFI9ypgrywDnZrTF-r1-2:-N-Yr1yl8PZ09nS4Lyv7:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c6a0f38510702774cfa7f85812f90179b2fa3a52/v21.1/frequently-asked-questions.md#L2)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

```suggestion
title: CockroachDB FAQs
```
</blockquote></details>

Done.
___
*[v22.1/frequently-asked-questions.md, line 2 at r1](https://reviewable.io/reviews/cockroachdb/docs/13596#-N-YLFI9ypgrywDnZrTG-r1-2:-N-Yr2N22fjj455UG0Vr:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/c6a0f38510702774cfa7f85812f90179b2fa3a52/v22.1/frequently-asked-questions.md#L2)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

```suggestion
title: CockroachDB FAQs
```
</blockquote></details>

Done.

**Comment by stbof** [comment]
LGTM.

**Comment by lnhsingh** [comment]
TFTR!

**Pull request merged**